stdlib/bytesLabels.mli

```diff
@@ -161,6 +161,12 @@ val mapi : f:(int -> char -> char) -> bytes -> bytes
     index (in increasing index order) and stores the resulting bytes
     in a new sequence that is returned as the result. *)
 
+val filter : p:(char -> bool) -> bytes -> bytes
```

Comment:
Shouldn't the predicate be labeled `f` to match the other labeled `filter` functions?

Reply:
Thanks, fixed.